FBSimulatorControl/Configuration/FBSimulatorControlConfiguration.m

```diff
@@ -57,6 +64,7 @@ - (BOOL)isEqual:(FBSimulatorControlConfiguration *)object
     return NO;
   }
   return [self.simulatorApplication isEqual:object.simulatorApplication] &&
+         ((self.deviceSetPath == nil && object.deviceSetPath == nil) || [self.deviceSetPath isEqual:object.deviceSetPath]) &&
```

Comment:
Could be simplified to `(self.deviceSetPath == object.deviceSetPath) || [.. isEqual:]`